cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -1653,7 +1653,7 @@ function Install-Pigz {
 # TODO(pjh): move the logging agent code below into a separate
 # module; it was put here temporarily to avoid disrupting the file layout in
 # the K8s release machinery.
-$LOGGINGAGENT_VERSION = '1.7.3'
+$LOGGINGAGENT_VERSION = '1.7.5'
```

Comment:
Not necessarily for this PR but do these installers come with a checksum that we can check when we download the file around line 1759?

Reply:
https://github.com/kubernetes/node-problem-detector/pull/545 introduces SHA-512 checksums next to each NPD tarball starting with 0.8.8.